### PR TITLE
chore: move knobs decorator to main.js to fix web crashes

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

**Comment by deorukhkarchaitanya** [comment]
@susheelachoudry web build won't crash after this

File: storybook/native/main.js [hunk 1]
```diff
@@ -31,6 +32,8 @@ addDecorator((Story) => (
   </SpaceAround>
 ));
 
+addDecorator(withKnobs);
```
Comment (susheelachoudry) [inline storybook/native/main.js]: could you add it in the preview Js too for web

Comment (deorukhkarchaitanya) [reply storybook/native/main.js]: Done

**Comment by susheelachoudry** [comment]
lgtm

**Pull request merged**